temba/ussd/tests.py

```diff
@@ -443,3 +444,41 @@ def test_interrupt(self, create_incoming):
         session = USSDSession.objects.get()
         self.assertIsInstance(session.ended_on, datetime)
         self.assertEqual(session.status, USSDSession.INTERRUPTED)
+
+    def test_ussd_subflow(self):
+        # start a session
+        callback_url = reverse('handlers.vumi_handler', args=['receive', self.channel.uuid])
+        ussd_code = "*111#"
+        data = dict(timestamp="2016-04-18 03:54:20.570618", message_id="123456", from_addr="+250788383383",
+                    content="None", transport_type='ussd', session_event="new", to_addr=ussd_code)
+        response = self.client.post(callback_url, json.dumps(data), content_type="application/json")
+        child_flow = self.get_flow('ussd_subflow_child')
+        flow = self.get_flow('ussd_subflow_parent', dict(NEW_CONTACT_FLOW_ID=child_flow.uuid))
```

Comment:
If these two flows are in the same import file, then this sort of mapping is unnecessary. They'll get new uuids, relatively replaced during the flow import itself.